src/settings/base.py

```diff
@@ -222,6 +223,30 @@
 CELERY_TIMEZONE = 'UTC'
 CELERY_WORKER_PREFETCH_MULTIPLIER = 1
 
+# ============================================================================
+# Caching
+# ============================================================================
+CACHES = {
+    'default': {
+        'BACKEND': 'redis_cache.RedisCache',
+        'LOCATION': [os.environ.get("REDIS_URL", "redis://redis:6379")],
+        'OPTIONS': {
+            'PARSER_CLASS': 'redis.connection.HiredisParser',
+            'CONNECTION_POOL_CLASS': 'redis.BlockingConnectionPool',
+            'CONNECTION_POOL_CLASS_KWARGS': {
+                'max_connections': 30,
+                'timeout': 20,
+            },
+            'MAX_CONNECTIONS': 30,
+            'PICKLE_VERSION': -1,
+        },
+    },
+}
+REST_FRAMEWORK_EXTENSIONS = {
+    'DEFAULT_CACHE_RESPONSE_TIMEOUT': 60 * 15,
+    'DEFAULT_USE_CACHE': 'default'
```

Comment:
oh, also, do we need this?

Reply:
Oh nope, it will use default automatically.
```suggestion
```